### ISSUE TITLE
Table data layout question

### ISSUE THREAD

**Issue by BarabasGitHub** [tiehuis/zig-ryu#1]
I was looking through the code and I actually wonder about these tables. 
https://github.com/tiehuis/zig-ryu/blob/724f66d81cf598d53df8eca2a93bf2eb8c7cdb1d/src/ryu64_full_table.zig#L18-L20

Does the ``[][]u64`` notation mean that it goes through two pointers and that each row can be a different length? If so, wouldn't it then be better to have a [][2]u64 table? Or does it already compile down to that?

**Comment by tiehuis** [comment]
Yeah I did think that when I did those tables. A quick test shows the code doesn't compile to the same code but I'll have a closer look and come back here before changing it.

**Comment by BarabasGitHub** [comment]
Yeah I was thinking that if it has to do an extra pointer hop it might
explain part of the performance difference between the zig and C
implementation.

Op vr 10 aug. 2018 14:12 schreef Marc Tiehuis <notifications@github.com>:
